### DISCUSSION TITLE
How do I make the "Figure X." part of the figure or table caption bold, without making the rest of the caption bold?

### DISCUSSION THREAD

**Discussion by EricKrantz** [quarto-dev/quarto-cli#11457] [Q&A]
### Description

This question was asked on StackOverflow on Oct 17, 2022, and has yet to receive an answer that works:

https://stackoverflow.com/questions/74097579/make-figure-text-bold-in-quarto-figure-captions

The question regards making a figure caption and number bold, without making the rest of the figure caption text bold. This seems to be a common style in publications, but I'm assuming it can't easily be achieved in Quarto because the question on SO has over 4000 views and no good answer. Cross-posting to bring attention to the question. 

Here is an example of what we would like to do:

**Figure 3.** A very nice figure to look at.

**Comment by mcanouil** [comment]
Duplicate of:

- https://github.com/quarto-dev/quarto-cli/discussions/3273

Let's try to avoid multiplying discussions about the same thing with the same contributors and the same answers.

Note that the answer/solution (if it exists) highly depends on the format.

**Reply by EricKrantz** [reply]
I guess I didn't search "lua filter", and wouldn't have known to, having never heard of a lua filter. 

My question simplified the overly complex question that you cited, which is barely searchable, overly detailed, and contains irrelevant code. Perhaps that's why it hasn't gained a sufficient answer yet. 

Also, I was clear that I wanted to bring attention to the question, hence the post. Why would you close it immediately?

**Reply by mcanouil** [reply]
Because the question has several answers (on Slack and the discussions I have linked) that you may not understand or like but nonetheless are quite valid answers.

There are no straight way to do what you want right now.
And anyway, it's completely dependent of the chosen format.

> My question simplified the overly complex question that you cited, which is barely searchable, overly detailed, and contains irrelevant code. Perhaps that's why it hasn't gained a sufficient answer yet.

How can you tell the code is irrelevant if you don't even know what it is?

**Reply by gordonwoodhull** [reply]
FWIW I think the Lua filter in #3273 is fine and should work across formats.

Since this is generated output, post-processing is the only choice. 

I suppose it could be generalized by reading the `fig-title` instead of hard-coding "Figure".

**Comment by mcanouil** [comment]
For reference, the suggestion to have a way to have bold labels:
- https://github.com/quarto-dev/quarto-cli/issues/1095

**Reply by mcanouil** [reply]
And

- https://github.com/quarto-dev/quarto-cli/discussions/7386
